cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+import numpy as np
+from cleanlab.filter import _find_multilabel_issues_per_class
+from cleanlab.internal.multilabel_utils import get_onehot_num_classes
+from collections import defaultdict
+
+
+def common_multilabel_issues(
+    labels=list,
+    pred_probs=None,
+    *,
+    class_names=None,
+    confident_joint=None,
+) -> pd.DataFrame:
+    """Returns the Tags that are often mis-labeled.
+
+
+    This method provides two scores in the Pandas DataFrame that is returned:
+
+    * **num_examples**: The number of examples where the given/predicted label differ.
+    * **Issue_probability**: `(num_examples / total number of examples in the dataset`).
```

Comment:
```suggestion
```

Reply:
this is redundant with the documentation in the return type which is much clearer